harness/determined/load/_load_trial_controller.py

```diff
@@ -147,15 +147,15 @@ def prepare_tensorboard(
         env, env.experiment_config["checkpoint_storage"], container_path
     )
     try:
-        from determined.tensorboard.metric_writers import pytorch
+        from determined.tensorboard.metric_writers import tensorflow
```

Comment:
non-blocking: as mentioned just make sure that this works well in containers without tensorflow installed.

Reply:
Checked and it works 🥳